### PR TITLE
Keratin/Tailwind merge

### PR DESCRIPTION
**Pull request by PixelJanitor**

### CODE REVIEW

File: app/components/build-approval-button.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default Component.extend({
   isDisabled: false,
   'aria-label': 'Approve this build',
   tagName: 'button',
```
Comment (PixelJanitor) [inline app/components/build-approval-button.js]: test multi line comment

File: app/components/build-approval-button.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ export default Component.extend({
...
-  classNames: ['build-approval-button btn btn-md btn-success ml-1 px-2 flex items-center'],
+  classNames: ['build-approval-button btn-success ml-1 flex items-center'],
```
Comment (PixelJanitor) [inline app/components/build-approval-button.js]: Test for an ovelapping comment

File: app/styles/app.css.foobar [hunk 3]
```diff
@@ -1,6 +1,6 @@
 @charset "UTF-8";
```
Comment (djones) [inline app/styles/app.css.foobar]: filename has `.foobar`?

File: app/styles/base/base.css [hunk 4]
```diff
@@ -52,7 +52,7 @@ a > code {
 
 b,
 strong {
-  font-weight: var(--font-weight-strong, var(--keratin-font-weight-strong));
+  @apply font-semibold;
```
Comment (djones) [inline app/styles/base/base.css]: nice

**Pull request merged**